PyPDF2/_page.py

```diff
@@ -1433,9 +1439,12 @@ def process_operation(operator: bytes, operands: List) -> None:
         return output
 
     def extract_text(
-        self, Tj_sep: str = "", TJ_sep: str = "", space_width: float = 200.0
```

Comment:
It's pretty unfortunate, but please leave those two parameters in there; maybe with the default `None`. Then add a deprecation notice if they don't have the default value.

It is ok that they will no longer do anything, but I don't want people to have exceptions (I'll add a deprecation note in the commit + in the CHANGELOG)

The new parameter orientations should be at the end if anybody uses positional arguments.

Reply:
Would it be ok to put them at the end considering they will be used only as named parameters?

Reply:
How do you know that nobody used it as positional parameters?

Reply:
What about this implementation:
```
    def extract_text(
        self,
        *args,
        Tj_sep: str = None, TJ_sep: str = None,
        orientations: Union[int, Tuple[int, ...]] = (0, 90, 270, 360),
        space_width: float = 200.0,
    ) -> str:
        if len(args)>=1:
            if isinstance(args[0],(str,list)):
                warnings.warn("parameters Tj_Sep, TJ_sep depreciated, and will be removed in PyPDF2 3.0.0.",DeprecationWarning)
                Tj_sep =args[0]
                if len(args)>=2 and isinstance(args[1],str):
                    TJ_sep =args[1]
                if len(args)>=3 and isinstance(args[2],(list,int)):
                    orientations = args[2]
            elif isinstance(args[0],(list,int)):
                orientations = args[0]
        if isinstance(orientations, int):
            orientations = (orientations,)
        return self._extract_text(
            self, self.pdf, orientations, space_width, PG.CONTENTS
        )
```

We will have both new and old version 😊